bounter/bounter.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Author: Filip Stefanak <f.stefanak@rare-technologies.com>
+# Copyright (C) 2017 Rare Technologies
+#
+# This code is distributed under the terms and conditions
+# from the MIT License (MIT).
+
+from .count_min_sketch import CountMinSketch, CardinalityEstimator
+from bounter_htc import HT_Basic as HashTable
+
+
+def bounter(size_mb=64, need_iteration=True, need_counts=True, log_counting=None):
+    """Factory method for bounter implementation.
+
+    Args:
+            size_mb (int): Desired memory footprint of the counter. Defaults to 64MB
```

Comment:
I'd even remove the default, to force people to make an explicit choice. This is what separated Bounter from Counter.